src/scala/com/github/johnynek/bazel_deps/DepsModel.scala

```diff
@@ -419,7 +440,7 @@ case class ProjectRecord(
 
   // Cache this
   override lazy val hashCode: Int =
```

Comment:
this one is only here because it transitively depends on a lot and we want to not recompute all the time (and we know that these are used as hash keys.